setup.py

```diff
@@ -37,7 +39,7 @@
 
 setup(
     name='djangocms-admin-style',
-    version='3.1.1',
+    version=__version__,
```

Comment:
We don't need this. This package uses bumpversion to change version automatically.

Reply:
Thanks for the hint!